pyccel/ast/core.py

```diff
@@ -661,16 +664,16 @@ def unravelled(self):
     def lhs(self):
         return self.body[-1].lhs
 
-    def insert2body(self, obj, back=True):
+    def insert2body(self, *obj, back=True):
         """ Insert an object to the body of the codeblock
```

Comment:
I think this now support inserting many objects not only one, can you update this desc

Reply:
Fixed in 9a158fcc